recipes/tesseract/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "tesseract" %}
+{% set version = "3.05.01" %}
+{% set sha256 = "05898f93c5d057fada49b9a116fc86ad9310ff1726a0f499c3e5211b3af47ec1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tesseract-ocr/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  host:
+    - gcc       # [linux]
+    - llvm      # [osx]
+    - libpng
+    - jpeg
+    - libtiff
+    - automake
+    - autoconf
+    - libtool
+    - zlib
+    - leptonica
+  run:
+    - libgcc    # [linux]
```

Comment:
This is automatically added by the `compiler` syntax noted above. So we don't need to add it explicitly. IOW please go ahead and drop this.